packages/GA4Client/src/common.js

```diff
@@ -132,6 +132,15 @@ Common.prototype.standardizeParameters = function (parameters) {
 };
 
 Common.prototype.standardizeName = function (name) {
+    try {
+        name = window.GoogleAnalytics4Kit.setCustomNameStandardization(name);
+    } catch (e) {
+        console.warn(
+            'Error calling setCustomNameStandardization callback. Check your callback.  Data will still be sent without user-defined standardization',
```

Comment:
Should we provide a link to our docsite? If not in the error log, perhaps we can provide a link to the docsite or example code as a comment?

Reply:
Haven't started the docs yet.  But that's a good idea.